app/views/home/index.html.erb

```diff
@@ -1 +1,6 @@
-<%= render partial: "signup_form" %>
+<h1>￼Connecting businesses, communities and the tech talent pipeline</h1>
+
+<div class="links">
+  <%= link_to "Learn More", learn_more_index_path %>
+  <%= link_to "Sign Up", signup_path %>
+</div>
```

Comment:
These paths will also need to be updated once we move back to the static pages controller